test/integration/tests/abrmd_policycphash.sh

```diff
@@ -23,7 +23,7 @@ cleanup "no-shutdown"
 
 generate_policycphash() {
     tpm2 startauthsession -S session.ctx -g sha256
-    tpm2 policycphash -S session.ctx -L policy.cphash --cphash cp.hash
+    tpm2 policycphash -S session.ctx -L policy.cphash --cphash-input cp.hash
```

Comment:
keep one of these the old deprecated option just for test coverage

Reply:
fixed